add paying reviewers page to sidebar (#764)
actualbudget/docs: docs/contributing/leadership/funding.md

## Accepted Expenses

> Actual’s Open Collective funds should be used to cover operational costs directly related to the project. This includes, but is not limited to domain / hosting costs and code signing certificates. Please go through the submission process below _before_ making any purchases.

Reviewer: ```suggestion
Actual’s Open Collective funds should be used to cover operational costs directly related to the project. This includes, but is not limited to, domain/hosting costs, as well as code signing certificates. Please review the submission process below _before_ making any purchases.
```
